To: vim_dev@googlegroups.com
Subject: Patch 7.3.463
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.463
Problem:    When using ":s///c" the cursor is moved away from the match.
	    (Lawman)
Solution:   Don't move the cursor when do_ask is set. (Christian Brabandt)
Files:	    src/ex_cmds.c


*** ../vim-7.3.462/src/ex_cmds.c	2012-02-22 14:58:24.000000000 +0100
--- src/ex_cmds.c	2012-03-07 14:54:21.000000000 +0100
***************
*** 5151,5160 ****
  
  	if (!global_busy)
  	{
! 	    if (endcolumn)
! 		coladvance((colnr_T)MAXCOL);
! 	    else
! 		beginline(BL_WHITE | BL_FIX);
  	    if (!do_sub_msg(do_count) && do_ask)
  		MSG("");
  	}
--- 5151,5163 ----
  
  	if (!global_busy)
  	{
! 	    if (!do_ask)  /* when interactive leave cursor on the match */
! 	    {
! 		if (endcolumn)
! 		    coladvance((colnr_T)MAXCOL);
! 		else
! 		    beginline(BL_WHITE | BL_FIX);
! 	    }
  	    if (!do_sub_msg(do_count) && do_ask)
  		MSG("");
  	}
*** ../vim-7.3.462/src/version.c	2012-02-29 19:19:57.000000000 +0100
--- src/version.c	2012-03-07 14:55:39.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     463,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
151. You find yourself engaged to someone you've never actually met,
     except through e-mail.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
